docs/source/geopmsession.1.rst

```diff
@@ -46,14 +47,15 @@ Description
 -----------
 
 Command line interface for the GEOPM service batch read features. The input to
-the command line tool has one request per line. A request for reading is made
-up of three strings separated by white space. The first string is the signal
-name, the second string is the domain name, and the third string is the domain
-index.
+the command line tool has one request per line. A request for reading is made up
+of three strings separated by white space. The first string is the signal name,
+the second string is the domain name, and the third string is the domain index.
+Provide the "``*``" character as the third string to request all domains
+available on the system.
 
-Unless the ``-n`` option is specified, a descriptive header is written
-first and then the output from reading the requests.  Unless ``-p`` is
-specified, only one line of output after the header is printed.
+Unless the ``-n`` option is specified, a descriptive header is written first and
+then the output from reading the requests.  Unless ``-p`` is specified, only one
```

Comment:
This is confusing as-is.  Suggestion:
```suggestion
If the ``-n`` option is specified a descriptive header is written first and
then the output from reading the requests.  Unless ``-p`` is specified, only one
```

Reply:
The -n option skips writing the CSV header.  I'm not sure how to make that clearer, I think the other order is more confusing?

"A descriptive header is written first, unless the ``-n`` option is specified in which the header is not printed.  The output from reading the requests is then written."

Reply:
I had understood the behavior of -n backwards.  I think it's fine.

The only other language I'd consider is "To suppress the printing of the CSV column headers, use -n".